app/main.py

```diff
@@ -1,6 +1,40 @@
+import os
+
+import requests
+
+BASE_URL = "http://api.weatherapi.com/v1"
+API_KEY = os.environ.get("API_KEY")
+
+FILTERING_CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    if not API_KEY:
+        raise Exception("API_KEY environment variable is not set")
+
+    response = requests.get(
+        url=f"{BASE_URL}/current.json",
+        params={
+            "key": API_KEY,
+            "q": FILTERING_CITY,
+        },
+    )
+    response.raise_for_status()
+
+    data = response.json()
```

Comment:
To co się tu dzieje zależy od zewnętrznego wystawcy api, więc się podatne na różnego rodzaju błędy, timeouty i inne errory. Powinniśmy złapać tą logikę w blok try/except. Warto też wyciągnąć tą logikę do osobnej funkcji pod nazwą np. `do_request()`